python/cudf/cudf/tests/test_contains.py

```diff
@@ -72,3 +74,21 @@ def test_index_contains(values, item, expected):
 def test_rangeindex_contains():
     assert_eq(True, 9 in RangeIndex(start=0, stop=10, name="Index"))
     assert_eq(False, 10 in RangeIndex(start=0, stop=10, name="Index"))
+
+
+@pytest.mark.parametrize("dtype", SCALAR_TYPES)
```

Comment:
Need to convert this to a list otherwise we don't get ordering guarantees and pytest-xdist isn't happy:
```suggestion
@pytest.mark.parametrize("dtype", list(SCALAR_TYPES))
```

Reply:
Sorry, gave poor guidance previously. Probably want to be consistent with how we're doing it elsewhere.

Reply:
Bool needed it's own test due to another bug, so I had to separate all the tests out and ended up not needing this at all.